src/akeneo/harness.yml

```diff
@@ -30,6 +30,17 @@ attributes:
     user: akeneo
     pass: akeneo
     name: akeneo
+    import:
+      steps:
+        - passthru "echo \"Importing database file\""
+        - run "set -e -o pipefail; zcat \"$1\" | mysql -h \"$DB_HOST\" -u root -p\"$DB_ROOT_PASS\" \"$DB_NAME\""
```

Comment:
Missing the `pv` pipe via passthru - important for large databases.
https://github.com/inviqa/harness-base-php/blob/0.3.x/src/_base/docker/image/console/root/lib/task/assets/apply.sh#L28 has an example

Reply:
Assumption of root user will conflict with https://github.com/inviqa/harness-base-php/pull/197

Reply:
Yes some confusion there, I think there is more work to be done for #197 as there are still scripts in the harnesses using DB_ROOT_PASS, either that or I somehow don't have these changes.... I will check and remove DB_ROOT_PASS and convert anything else as per #197

Reply:
Seems just my confusion.... Andy left in DB_ROOT_PASS I guess for backwards compatibility and misread this